src/cli/format.js

```diff
@@ -254,7 +283,7 @@ async function formatStdin(context) {
     : path.relative(process.cwd(), filepath);
 
   try {
-    const input = await getStdin();
+    const input = (await getStdin()) || (await getInputFromTerminal());
```

Comment:
I prefer add another parameter to this function. Use `getInputFromTerminal` only if `--stdin` flag found.

Reply:
Will do, my initial approach was using a flag but went with this one for simplicity.
Will add the flag

Reply:
add a new cli option `--std-in`